great_expectations/datasource/data_connector/file_path_data_connector.py

```diff
@@ -92,6 +92,11 @@ def _get_data_reference_list_from_cache_by_data_asset_name(
             )
         )
 
+        if len(self.sorters) > 0:
+            batch_definition_list = self._sort_batch_definition_list(
+                batch_definition_list=batch_definition_list
+            )
```

Comment:
I might be missing something here, and I know `self._sort_batch_definition_list` was already implemented, but in the case where there is more than one sorter in `self.sorters` would we want to pass the sorter into these function so we know which one to use?

Reply:
If Sorting is a feature that a lot of people end up using, what you described is a great candidate for a future refactor.  Currently the DataConnector will use *all* sorters that are configured in the order they are configured... so `self._sort_batch_definition_list()` just goes through `self.sorters` and applies them one by one. No need to pass in a specific one, just need to know there is something there (`len(self.sorters) > 0`)